src/main/java/me/itzg/helpers/users/ManageUsersCommand.java

```diff
@@ -223,22 +228,30 @@ private JavaUser resolveJavaUserId(SharedFetch sharedFetch, List<? extends JavaU
 
             })
             .orElseGet(() -> {
-
+                JavaUser finalUser = null;
```

Comment:
I think it'll be helpful (for future me) to add an inline comment here explaining why the user found in the cache can't be immediately returned.